woocommerce/woocommerce-android
# [Simplified login] Update design for the Jetpack Installation initial screen

Closes: #7949 

### Description
This just updates the design of the Jetpack Installation initial screen.

### Testing instructions
1. Open site picker.
2. Click on "Add a store" then "Connect an existing store"
3. Enter site address of a non-jetpack site.
4. Confirm the screen looks as expected, and your site's favicon is loaded.

### Images/gif
| Scenario | Light | Dark |
| --- | --- | --- |
| Install | ![image](https://user-images.githubusercontent.com/1657201/203943724-7e545e50-c3d0-4d33-ace6-6f161e6a89fa.png) | ![image](https://user-images.githubusercontent.com/1657201/203943792-775d1e01-e6c5-492d-b79a-e568cd650ce4.png) |
| Connect | ![image](https://user-images.githubusercontent.com/1657201/203943907-0b6b901d-266b-4d08-831c-35351e324a6e.png) | ![image](https://user-images.githubusercontent.com/1657201/203943950-85d1b585-089e-4236-9abd-1a20b00e3887.png) |
